pkg/cmd/pulumi/packagecmd/package_add.go

```diff
@@ -88,6 +90,10 @@ extension, Pulumi package schema is read from it directly:
 
 			pkg, err := SchemaFromSchemaSource(pctx, plugin, parameters)
 			if err != nil {
+				var diagErr hcl.Diagnostics
+				if errors.As(err, &diagErr) {
+					return fmt.Errorf("failed to get schema.  Diagnostics: %w", errors.Join(diagErr.Errs()...))
```

Comment:
I wish we were consistent about double spaces after periods or not

Reply:
Old habbits.  It should probably be a single space.
```suggestion
					return fmt.Errorf("failed to get schema. Diagnostics: %w", errors.Join(diagErr.Errs()...))
```

Reply:
std lib had the same issue, they went with single spaces everywhere: https://go-review.googlesource.com/c/go/+/20022